migrations/20221031220419_alter_request_add_indexes.js

```diff
@@ -0,0 +1,31 @@
+const REQUEST_TABLE_NAME = 'request'
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+export async function up(knex) {
+  await knex.schema.alterTable(REQUEST_TABLE_NAME, (table) => {
+    table.index(['stream_id'])
+    table.index(['status'])
```

Comment:
We shouldn't need this since we have the multicolumn indexes that start with status below